app/javascript/lib/imperial.js

```diff
@@ -1336,31 +1356,46 @@ export default class Imperial {
         return;
       }
       case 'factory': {
-        // If nation cannot afford to build a factory
-        if (this.nations.get(this.currentNation).treasury < 5) {
-          if (this.variant === 'withoutInvestorCard') {
```

Comment:
Players can now intentionally waste the nation's time not building a factory

Slightly useful if you know your hold over a nation is temporary and really want to screw them for 1 whole round